app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -248,14 +249,11 @@ def networks
 
   def sshkeys
     collector.sshkeys.each do |tkey|
-      tenant_key = {
-        :creationDate => tkey.creation_date,
-        :name         => tkey.name,
-        :sshKey       => tkey.ssh_key,
-      }
-
-      # save the tenant instance
-      persister.auth_key_pairs.build(:name => tenant_key[:name])
+      persister.auth_key_pairs.build(
+        :name        => tenant_key[:name])
+        :sshKey      => tkey.ssh_key,
```

Comment:
There is no `sshKey` column on the authentications table, I think you want `:public_key` here